src/components/CategoriesView.jsx

```diff
@@ -0,0 +1,29 @@
+import "./CategoriesView.css";
+import { Link } from "react-router-dom";
+import MediumCardCollection from "./MediumCardCollection";
+
+export default function CategoriesView(props) {
+  let { CategoryType, CategoryPlaylists } = props;
```

Comment:
In this project, and in most, usually lowercase first letters are used for hook variable names.